src/components/charts/AddVisualizationForm.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import { useTranslation } from 'react-i18next';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import Button from '@material-ui/core/Button';
+import { List } from 'immutable';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    display: 'flex',
+    justifyContent: 'center',
+    margin: theme.spacing(2),
+  },
+  subcontainer: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  formControl: {
+    margin: theme.spacing(1),
+    width: '100%',
+  },
+  select: {
+    padding: 0,
+    width: '300px',
+  },
+  inputLabel: {
+    backgroundColor: '#F7F7F7',
+  },
```

Comment:
you're right, in this case it doesn't make a difference, because the text in the `<InputLabel>` is short. but see here why it does make a difference: https://github.com/graasp/graasp-insights/pull/72#discussion_r519698356. so I'll keep it just in case this text becomes say _'Dataset (Required)'_

Reply:
Oh that was caused by the label prop of the select being only `Dataset` while the `InputLabel` value was `Dataset (required)`. So if we match both of them to `Dataset (required)` there shouldn't be any issues. This is some weird behavior but I guess they just check the length of the label prop for the hiding parts of the bounding box and display the value of the `InputLabel`

Reply:
ah nice catch. did not see your response to that comment. you're right, fixed it